From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fredric Silberberg <fredric@fsilberberg.com>
Date: Wed, 19 Nov 2014 14:20:53 -0500
Subject: [PATCH 0494/6262] Removed the tail command

The build scripts were still calling the tail command for following the log
file, even though we're now using netconsole. I've removed them.

Change-Id: I48498c1ef338f99130e447097081db92b394e1aa
---
 .../src/main/resources/cpp-zip/ant/build.properties    |  3 +--
 .../src/main/resources/cpp-zip/ant/build.xml           |  5 -----
 .../src/main/resources/java-zip/ant/build.properties   |  2 --
 .../src/main/resources/java-zip/ant/build.xml          | 10 ----------
 4 files changed, 1 insertion(+), 19 deletions(-)

diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.properties b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.properties
index c409af4cc317364f6c04560dc61faf79d710233f..3032388f2cf03d3ff8607742871eee12015f1b6e 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.properties
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.properties
@@ -3,7 +3,6 @@ username=admin
 password=
 deploy.dir=/home/lvuser
 deploy.kill.command=/usr/local/frc/bin/frcKillRobot.sh -t -r
-deploy.log.file=/var/local/natinst/log/FRC_UserProgram.log
 command.dir=/home/lvuser/
 
 # Libraries to use
@@ -14,4 +13,4 @@ roboRIOAllowedImages=18
 # Ant support
 wpilib.ant.dir=${wpilib}/ant
 jsch.jar=${wpilib.ant.dir}/jsch-0.1.50.jar
-classloadertask.jar=${wpilib.ant.dir}/ant-classloadertask.jar
\ No newline at end of file
+classloadertask.jar=${wpilib.ant.dir}/ant-classloadertask.jar
diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
index fc22f0d8bed5030f9fb34ff3af3061980eb1cae7..1f2b051f0dacf02878407c0e9af94534b1b17ce7 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml
@@ -57,11 +57,6 @@
        trust="true"
        command=". /etc/profile.d/natinst-path.sh; chmod a+x ${deploy.dir}/${out}; ${deploy.kill.command};"/>
 
-    <sshexec host="${target}"
-       username="${username}"
-       password="${password}"
-       trust="true"
-       command="tail -F -n 0 ${deploy.log.file}"/>
   </target>
   
   <target name="kill-program" depends="get-target-ip" description="Kill the currently running FRC program">
diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
index b259e7fa85a435394219df24e2504627eca606fb..cb8d6e363db6ddc2f0b4c0b7bd9c23398b3a8697 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties
@@ -3,8 +3,6 @@ username=admin
 password=
 deploy.dir=/home/lvuser
 deploy.kill.command=. /etc/profile.d/natinst-path.sh; /usr/local/frc/bin/frcKillRobot.sh -t -r
-deploy.log.file=/var/local/natinst/log/FRC_UserProgram.log
-deploy.log.command=tail -F -n 0 ${deploy.log.file}
 debug.flag.dir=/tmp/
 debug.flag.command=chown lvuser:ni ${debug.flag.dir}frcdebug
 command.dir=/home/lvuser/
diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml
index d457e689c628fbdbdf9131c8fcf8d78c2ef6ff14..7d2bc6d202f6c0600b847612a1d2de0225149e86 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml
@@ -97,11 +97,6 @@
        trust="true"
        command="${deploy.kill.command};"/>
 
-    <sshexec host="${target}"
-       username="${username}"
-       password="${password}"
-       trust="true"
-       command="${deploy.log.command}"/>
   </target>
 
   <target name="debug-deploy" depends="jar,get-target-ip,dependencies" description="Deploy the jar and start the program running.">
@@ -124,11 +119,6 @@
         trust="true"
         command="${deploy.kill.command}"/>
 
-    <sshexec host="${target}"
-        username="${username}"
-        password="${password}"
-        trust="true"
-        command="${deploy.log.command}"/>
   </target>
 
   <!-- Simulate -->
